test/node.js

```diff
@@ -328,6 +334,224 @@ node.randomPassword = function () {
 	return Math.random().toString(36).substring(7);
 };
 
+
+// Init whole application inside tests
+node.initApplication = function (cb) {
+	var modules = [], rewiredModules = {};
+	// Init dummy connection with database - valid, used for tests here
+	var options = {
+	    promiseLib: Promise
+	};
+	var pgp = require('pg-promise')(options);
+	node.config.db.user = node.config.db.user || process.env.USER;
+	var db = pgp(node.config.db);
+
+	// Clear tables
+	db.task(function (t) {
+		return t.batch([
+			t.none('DELETE FROM blocks WHERE height > 1'),
+			t.none('DELETE FROM blocks'),
+			t.none('DELETE FROM mem_accounts')
+		]);
+	}).then(function () {
+		// Force rewards start at 150-th block
+		node.constants.rewards.offset = 150;
+
+		var logger = {
+			trace: sinon.spy(),
+			debug: sinon.spy(),
+			info:  sinon.spy(),
+			log:   sinon.spy(),
+			warn:  sinon.spy(),
+			error: sinon.spy()
+		};
+
+		var modulesInit = {
+			accounts: '../modules/accounts.js',
+			transactions: '../modules/transactions.js',
+			blocks: '../modules/blocks.js',
+			signatures: '../modules/signatures.js',
+			transport: '../modules/transport.js',
+			loader: '../modules/loader.js',
+			system: '../modules/system.js',
+			peers: '../modules/peers.js',
+			delegates: '../modules/delegates.js',
+			multisignatures: '../modules/multisignatures.js',
+			dapps: '../modules/dapps.js',
+			crypto: '../modules/crypto.js',
+			// cache: '../modules/cache.js'
```

Comment:
Cache is disabled because is not used by current tests and require additional things to be loaded.